src/lib/Validator/Constraints/UserPasswordValidator.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Validator\Constraints;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\UserService;
+use eZ\Publish\API\Repository\Exceptions\InvalidArgumentException;
+use eZ\Publish\API\Repository\Values\User\User;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+
+/**
+ * Will check if logged user and password are match.
+ */
+class UserPasswordValidator extends ConstraintValidator
```

Comment:
Missing unit tests for `UserPasswordValidator`. Examples could be found here:  https://github.com/ezsystems/repository-forms/tree/master/tests/RepositoryForms/Validator/Constraints

Reply:
I added missing unit tests